### ISSUE TITLE
php_opcache.dll failed to load if installed in non default path (Windows)

### ISSUE THREAD

**Issue by hieblmedia** [appserver-io/appserver#612]
The php_opcache.dll path is wrong when the appserver is installed in non default path.
Tested in Windows 8.1.

```
Starting appserver.io...
Failed loading C:\Program Files\appserver\php\ext\php_opcache.dll
```

**Comment by wick-ed** [comment]
Hi @hieblmedia ,

issue is noted, we will have a look at it. :)

**Comment by wick-ed** [comment]
This will be fixed within upcoming release 1.0.1
